frontend/src/components/VThemeSelect/VThemeSelect.vue

```diff
@@ -26,14 +26,13 @@ const THEME_ICON_NAME = Object.freeze({
 const THEME_TEXT = {
   light: i18n.t(`theme.choices.light`),
   dark: i18n.t(`theme.choices.dark`),
+  system: i18n.t(`theme.choices.system`),
 }
 
-const colorMode = computed({
-  get: () => uiStore.colorMode,
-  set: (value) => {
-    uiStore.setColorMode(value)
-  },
-})
+const colorMode: Ref<ColorMode | undefined> = ref(undefined)
```

Comment:
```suggestion
const colorMode: Ref<ColorMode> = ref(uiStore.colorMode)
```
If the color mode is set in the UI store (if the user has a `ui.colorMode` cookie set), we can initialize with the correct value.